server/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -783,3 +753,39 @@ def _convert_into_model_endpoint_object(
                 endpoint_obj.status.drift_measures = drift_measures
 
         return endpoint_obj
+
+    @staticmethod
+    def _get_store_object(
+        project: str,
+    ) -> typing.Union[mlrun.model_monitoring.db.stores.base.store.StoreBase, None]:
+        """
+        Get the model endpoint store object.
+        Firstly trying to use project secret and if there is no such secret
+        it's trys to use the default/v3io store connection string.
+        Note : Use this method only for deleting/reading model endpoints.
+        """
+        try:
+            model_endpoint_store = (
+                server.api.crud.model_monitoring.helpers.get_store_object(
+                    project=project
+                )
+            )
+        except mlrun.errors.MLRunInvalidMMStoreType:
+            # TODO: delete in 1.9.0 - for BC trying to create default/v3io store
+            store_connection_string = (
+                mlrun.mlconf.model_endpoint_monitoring.endpoint_store_connection
+                or "v3io"
+                if mlrun.mlconf.is_ce_mode()
+                else None
+            )
```

Comment:
Could you please explain that condition?
`"v3io" if mlrun.mlconf.is_ce_mode()`?